### DISCUSSION TITLE
Authentication issues

### DISCUSSION THREAD

**Discussion by Teng3n** [Serene-Arc/bulk-downloader-for-reddit#846] [Q&A] [answered]
I am trying to authenticate my Reddit account. I run bdfr and it spits out a link to click. I click the link which takes me to a page to allow or deny. I click allow and it redirects to a localhost but breaks with ERR_CONNECTION_REFUSED. The script then stops and spits out Aborted! I've tried turning off Windows Firewall, but nothing seems to help. 

>python -m bdfr download .\Test --authenticate --upvoted --saved

**Comment by FowadIjaz** [comment, marked as answer]
You need to provide a ``--user username`` if you are using the ``--authenticate`` tag.
